tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3941,16 +3979,31 @@ <h2>Create a distance feature</h2>
         // Convert the data into the format expected by vz-bar-chart.
         // Stack models and labels so that each (model,label) is a separate
         // series
+        const lines = {};
         const mapped = {};
+        const self = this;
         _.forEach(data, (modelValues, modelInd) => {
           const obj = _.forEach(modelValues,
             (pointsForSeries, labelKey) => {
+              // Sort bars by their inference values for display.
               mapped[this.formatChartKey(labelKey, modelInd, data.length)] =
-                _.map(pointsForSeries, point => convertPoint(point, origKey));
+                _.map(pointsForSeries, point => convertPoint(
```

Comment:
Does this work as intended? When we have two models or multi-class classification features ("step") in each array will have different orders. Since bar-chart can have one name per bar, doesn't it just ignore the order of this array and assign an order itself so that the keys match? I am not 100% certain so it would be good if you could clarify!

Reply:
If you sort each array of bars by Y value before providing them to vz-bar-chart, then the created bar chart with sort the bars in a way where the bars go in order from tallest to shortest. It will use the ordering of the first series mainly, but if there are bars in other series that aren't in the first series, they will be appropriately ordered as well.